packages/otf/otf.0.1/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "2.0"
+name: "otf"
+version: "0.1"
+synopsis: "otf is a simple Output Test Framework"
+maintainer: "Olivier Levillain <olivier.levillain@telecom-sudparis.eu>"
+authors: "Olivier Levillain <olivier.levillain@telecom-sudparis.eu>"
+license: "BSD 3-Clause License"
+homepage: "https://gitlab.com/pictyeye/otf"
+bug-reports: "https://gitlab.com/pictyeye/otf/issues"
+depends: ["ocaml" "dune"]
+build: ["dune" "build" "-p" name]
```

Comment:
```suggestion
build: ["dune" "build" "-p" name "-j" jobs]
```
It would be more user friendly to allow users to set the number of jobs needed to build this package.

Reply:
No problem. However, I copy/pasted this from https://opam.ocaml.org/doc/Packaging.html (the "Creating a package definition file" section). Maybe you should consider updating this page?